libparsec/crates/client/src/certif/shamir_recovery_list.rs

```diff
@@ -331,3 +331,40 @@ pub async fn list_shamir_recoveries_for_others(
         })
         .await?
 }
+
+pub async fn get_shamir_recovery_share_data(
+    ops: &CertificateOps,
+    user_id: UserID,
+) -> Result<ShamirRecoveryShareData, CertifListShamirRecoveryError> {
+    ops.store
+        .for_read(|store| async move {
+            // TODO: check that the shamir is actually recoverable
```

Comment:
Meaning that enough recipient are still here / not revoked ?

Reply:
Exactly